client/views/admin/users/UserInfo.js

```diff
@@ -19,6 +19,8 @@ export function UserInfoWithData({ uid, username, ...props }) {
 	const showRealNames = useSetting('UI_Use_Real_Name');
 	const approveManuallyUsers = useSetting('Accounts_ManuallyApproveNewUsers');
 
+	const { reloadTable, ...otherProps } = props;
```

Comment:
I believe it's safe to do this destructuring at the function parameter declaration, i.e.:

```
export function AddUser({ roles, reloadTable, ...props }) {
```

I don't think `props` requires `reloadTable` property anywhere else.